Common/Orders/Fills/EquityFillModel.cs

```diff
@@ -112,7 +112,7 @@ public override OrderEvent LimitIfTouchedFill(Security asset, LimitIfTouchedOrde
                         if (askCurrent <= order.LimitPrice)
```

Comment:
>The LimitIfTouchedFill method should not fill using QuoteBar or Tick with TickType.Quote type

Just double checking, `LimitIfTouchedFill` is filling using quote data but getting triggered using trade data. The PR description describes another behavior?